src/directives/proxy.ts

```diff
@@ -45,17 +45,21 @@ export function getProxyDirective(args: GraphQLArgument[]) {
   }
 }
 
-function buildUri(uri: string, parentId?: string | number, argId?: string | number) {
+function buildUri(uri: string, parent?: any, args?: any) {
   let builtUri = uri
 
-  if (parentId) {
-    builtUri = builtUri.replace('$id', String(parentId))
-  }
-  if (argId) {
-    builtUri = builtUri.replace('$id', String(argId))
+  let env = Object.assign(parent, args);
```

Comment:
- Please use `const` if possible.
- `env` is ambiguous. Please use other name, such as `parentObj` or `parentWithArgs`.

```suggestion
  const parentWithArgs = Object.assign(parent, args);
```